deno_typescript/system_loader.js

```diff
@@ -4,96 +4,84 @@
 
 // @ts-nocheck
 /* eslint-disable */
-
-let System, __inst, __inst_s;
+let System, __ia, __i;
```

Comment:
Can we use more descriptive names than `__ia` and `__i`? I assume `__i` stands for instantiate? What is `__ia`?